config/locales/fr.yml

```diff
@@ -58,44 +57,75 @@ fr:
           help:
-            - Confirmer une identité avec un compte FranceConnect Auteur
+          - Confirmer une identité avec un compte FranceConnect Auteur
+        france_connect_uid:
+          help:
+          - Confirmer une identité avec un compte FranceConnect Signataire
         saml:
           help:
-            - Confirmer une identité avec un compte CSAM eID
+          - Confirmer une identité avec un compte CSAM eID
       decidim:
-        name: Omniauth Decidim
         explanation: Confirmer votre identité avec un compte Decidim externe
+        name: Omniauth Decidim
       france_connect:
-        name: France Connect
         explanation: Confirmer votre identité avec un compte FranceConnect
-      france_connect_uid:
-        name: FranceConnect Signataire
-        explanation: Confirmer votre identité avec un compte FranceConnect Signataire
+        name: France Connect
       france_connect_profile:
-        name: FranceConnect Auteur
         explanation: Confirmer votre identité avec un compte FranceConnect Auteur
+        name: FranceConnect Auteur
+      france_connect_uid:
+        explanation: Confirmer votre identité avec un compte FranceConnect Signataire
+        name: FranceConnect Signataire
       saml:
-        name: CSAM eID
         explanation: Confirmer votre identité avec un compte CSAM eID
+        name: CSAM eID
     devise:
       omniauth_registrations:
         new:
-          custom_agreement: "Je certifie être citoyen français ou résidant régulièrement en France, ainsi que l’exactitude des informations du profil"
+          custom_agreement: Je certifie être citoyen français ou résidant régulièrement
+            en France, ainsi que l’exactitude des informations du profil
           registration_info: |-
             Texte d'information pour la création de compte qui supporte le <code>HTML</code>.
             <br/><br/>
             <small>
               <i>Please change me with translation key <code>decidim.devise.omniauth_registrations.new.registration_info</code><i/>
             <small/>
+    event:
+      initiatives:
+        admin:
+          support_threshold_reached:
+            email_intro: decidim.event.initiatives.admin.support_threshold_reached.email_intro
+            email_outro: decidim.event.initiatives.admin.support_threshold_reached.email_outro
+            email_subject: decidim.event.initiatives.admin.support_threshold_reached.email_subject
+            notification_title: decidim.event.initiatives.admin.support_threshold_reached.notification_title
+    initiatives:
+      update:
+        error: fr.decidim.initiatives.update.error
+        success: fr.decidim.initiatives.update.success
+    omniauth:
+      france_connect:
+        external:
+          link: https://franceconnect.gouv.fr/
+          text: Qu'est-ce-que FranceConnect ?
+      france_connect_profile:
+        explanation: '<b>L''auteur d''une pétition ne peut être anonyme.</b><br/>
```

Comment:
Hi, 

Could you fix the double simple quotes in values like this one  for example ?
```
L''auteur d''une pétition ne peut être anonyme.
```

Thanks !

Reply:
To be more precise, I didn't check in local but I think they won't be escaped

Reply:
I have tested locally, I apologize it works well.

Seems good to me !